spec/player_spec.rb

```diff
@@ -0,0 +1,138 @@
+require_relative 'spec_helper'
+
+
+describe "Player" do
+  describe "#initialize" do
+    it "Takes a name" do
+      name = 'John'
+      player = Scrabble::Player.new(name)
+
+      player.must_respond_to :name
+      player.name.must_equal name
+      player.name.must_be_kind_of String
+
+      player.must_respond_to :plays
+      player.plays.must_equal []
+      player.plays.must_be_kind_of Array
+    end
+
+    it " should give you an empty array" do
+        player = Scrabble::Player.new(name)
+        player.must_respond_to :tiles
+        player.tiles.must_equal []
+        player.tiles.must_be_kind_of Array
+
+    end
+  end
+
+  describe "total_score" do
+    it "Sums one word" do
+      name = 'John'
+      player = Scrabble::Player.new(name)
+      player.play('hello')
+
+      total = player.total_score
+      total.must_equal 8
+      player.plays.length.must_equal 1
+    end
+
+    it "Sums multiple words" do
+      name = 'John'
+      player = Scrabble::Player.new(name)
+      player.play('hello')
+      player.play('bubbles')
+      player.play('wave')
+      total = player.total_score
+      total.must_equal 81
+    end
+
+    it "sums when no words have been played"  do
+      name = 'John'
+      player = Scrabble::Player.new(name)
+      count = player.total_score
+      count.must_equal 0
+    end
+  end
+
+  describe "play" do
+    it "plays word" do
+      name = 'John'
+      player = Scrabble::Player.new(name)
+      value = player.play('hello')
+
+      player.plays.length.must_equal 1
+      value.must_equal 8
+    end
+
+    it "plays after a player has won" do
+      name = 'John'
+      player = Scrabble::Player.new(name)
+      player.play('bubbles')
```

Comment:
Good test case! This is an important behavior for the game, but it would be an easy one to miss.